code/modules/overmap/sectors.dm

```diff
@@ -241,6 +251,27 @@ var/global/area/overmap/map_overmap
 	else
 		update_name()
 
+/// Is this a wrecked away site?
+/obj/effect/overmap/visitable/proc/is_wreck()
+	return FALSE
+
+/// Gets information on wrecked away sites, by setting `wreck_details` var.area
+/// Called once at init
+/obj/effect/overmap/visitable/proc/generate_wreck_details()
+	if(!is_wreck())
+		return
+
+	if(!wreck_details[WRECK_NAME])
+		wreck_details[WRECK_NAME] = name
+
+	if(!wreck_details[WRECK_AGE])
+		var/age = rand(1, max_age)
+		wreck_details[WRECK_AGE] = "Reconstruction of hull displacement suggests a high damage event occured approximately [age] years ago."
+
+	if(!wreck_details[WRECK_DAMAGE])
+		var/cause = pick("an equipment malfunction", "a meteor strike", "an electronic storm", "inter-vessel weaponry", "a collision")
+		wreck_details[WRECK_DAMAGE] = "Reconstruction indicates the damage was most likely cause by [cause]."
```

Comment:
Honestly really not a fan of a hull probe like that. There's no way to figure out any of that information with a probe just sampling some metal.

This information should be up to actual players and their chars to figure out, by exploring the away site or ship.

Like how would the probe even know if it's sampling a wreck or not? At what point does a ship or station become a wreck? It sounds really arbitrary.